prboom2/src/SDL/i_sound.c

```diff
@@ -227,6 +227,31 @@ static int addsfx(int sfxid, int channel, const unsigned char *data, size_t len)
   return channel;
 }
 
+static int getSliceSize(void)
+{
+  int limit, n;
+
+  if (snd_samplecount) return snd_samplecount;
```

Comment:
Please do the following:
```
if (snd_samplecount >= 32)
  return snd_samplecount * snd_samplerate / 11025;
```
so users can keep their config if they fine-tuned it with the previous implementation.

Reply:
Good idea. Since those users have likely moved away from 512, I'll put this condition too:
```
if ((snd_samplecount >= 32) && (snd_samplecount != 512)) 
  return snd_samplecount * snd_samplerate / 11025;
```

Reply:
No, please don't. We don't need two "di nothing else" cases.

Reply:
I admit that it's a hack, but shouldn't we handle the previous default? Or would it be safer to just do this for newer installs?